assets/css/index.styl

```diff
@@ -35,6 +35,13 @@ h1
 h2
   font-size: 1.5em
   margin: 0.83em 0
+  font-size: 22px;
+  line-height: 22px;
+  letter-spacing: 4px;
+  margin-bottom: 10px;
+  font-weight: 400;
+  text-transform: uppercase;
+  text-align: center;
```

Comment:
It doesn't really compete with the logo, but the letter spacing is weird. The spaces are so big it's almost the size of spaces between the words.